benchmarks/python/test_layernorm_bwd.py

```diff
@@ -91,9 +92,19 @@ def layernorm_bwd_fusion(
     fd.add_output(T28)
 
 
+def layernorm_bwd_iobytes(size: tuple, dtype: torch.dtype):
+    # Total IO bytes = in_tensor (size, dtype) + grad_out (size, dtype) + mean (size[0], float) +
+    #       invstd (size[0], float) + weights (size[1], dtype) +
+    #       grad_in (size, dtype) + grad_weights (size[1], dtype)+ grad_bias (size[1], dtype)
```

Comment:
Could you comment on how this might differ from the default computation in the eager case?